internal/controllers/aggregation/status.go

```diff
@@ -80,7 +88,7 @@ func (s *statusController) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	}
 
 	now := metav1.Now()
-	if ready {
+	if ready && maxReadyTime != nil {
```

Comment:
It's possible to have zero resources, which means ready would be true with a nil maxReadyTime